chcemvediet/apps/obligees/tests/test_forms.py

```diff
@@ -1,123 +1,219 @@
+                u'[@value="ccc"]'.format(url=reverse(u'obligees:autocomplete')))
+        self.assertIsNotNone(form_control)
+        details = html.find(u'.//div[@class="chv-obligee-widget-details "]')
+        street = details.find(u'span[@class="chv-obligee-widget-street"]')
+        self.assertEqual(street.text, u'ccc street')
+        zip = details.find(u'span[@class="chv-obligee-widget-zip"]')
+        self.assertEqual(zip.text, u'12345')
+        city = details.find(u'span[@class="chv-obligee-widget-city"]')
+        self.assertEqual(city.text, u'ccc city')
+        email = details.find(u'span[@class="chv-obligee-widget-email"]')
+        self.assertEqual(email.text, u'ccc@a.com')
 
-    @unittest.skip(u'FIXME')
     def test_new_form_with_initial_value_as_obligee_name(self):
         names = [u'aaa', u'bbb', u'ccc', u'ddd']
-        oblgs = [self._create_obligee(name=n) for n in names]
+        oblgs = [self._create_obligee(name=n, street=u'{} street'.format(n), city=u'{} city'.format(n), zip=u'12345', emails=u'{}@a.com'.format(n)) for n in names]
         form = self.Form(initial={u'obligee': u'ccc'})
         rendered = self._render(u'{{ form }}', form=form)
-        self.assertInHTML(u"""
-                <input class="autocomplete" data-autocomplete-url="{url}" id="id_obligee" name="obligee" type="text" value="ccc">
-                """.format(url=reverse(u'obligees:autocomplete')), rendered)
+        html = lxml.html.fromstring(rendered)
+        form_control = html.find(u'.//input'
+                                 u'[@class="form-control pln-autocomplete"]'
+                                 u'[@data-autocomplete-url="{url}"]'
+                                 u'[@name="obligee"]'
+                                 u'[@data-name="obligee"]'
+                                 u'[@type="text"]'
+                                 u'[@value="ccc"]'.format(url=reverse(u'obligees:autocomplete')))
+        self.assertIsNotNone(form_control)
+        details = html.find(u'.//div[@class="chv-obligee-widget-details "]')
+        street = details.find(u'span[@class="chv-obligee-widget-street"]')
+        self.assertEqual(street.text, u'ccc street')
+        zip = details.find(u'span[@class="chv-obligee-widget-zip"]')
+        self.assertEqual(zip.text, u'12345')
+        city = details.find(u'span[@class="chv-obligee-widget-city"]')
+        self.assertEqual(city.text, u'ccc city')
+        email = details.find(u'span[@class="chv-obligee-widget-email"]')
+        self.assertEqual(email.text, u'ccc@a.com')
 
-    @unittest.skip(u'FIXME')
     def test_submitted_with_empty_value_but_required(self):
         form = self.Form({u'obligee': u''})
         self.assertFalse(form.is_valid())
         self.assertEqual(form.errors[u'obligee'], [u'This field is required.'])
 
         rendered = self._render(u'{{ form }}', form=form)
         self.assertInHTML(u'<ul class="errorlist"><li>This field is required.</li></ul>', rendered)
```

Comment:
@martinmacko47 Je to jediny `assertInHtml` ktory ostal mam nahradit aj ten pre jednotvarnost alebo ho mozem nechat?

Reply:
@viliambalaz Ano, bolo by fajn to spravit rovnako ako v negativnom teste vedla. Tj. selectnut `errorlist = html.find(u'.//ul[@class="errorlist"]')` a pozriet, ze ma prave jeden `<li>` child so spravnym textom.

Reply:
:ok: